src/Resources/config/services.xml

```diff
@@ -19,5 +19,12 @@
             <tag name="kernel.event_listener" event="kernel.controller" method="onKernelController" priority="-1" />
         </service>
         <service id="apy_breadcrumb_trail.annotation.listener" alias="APY\BreadcrumbTrailBundle\EventListener\BreadcrumbListener" public="true" />
+
+        <service id="Doctrine\Common\Annotations\AnnotationReader" />
+
+        <service id="APY\BreadcrumbTrailBundle\EventListener\BreadcrumbListener">
+            <argument type="service" id="Doctrine\Common\Annotations\AnnotationReader" />
```

Comment:
Can you try getting this done with the autowiring that is enabled on this bundle?